administrator/language/de-DE/de-DE.com_plugins.ini

```diff
@@ -35,6 +35,7 @@ COM_PLUGINS_N_ITEMS_UNPUBLISHED="%d Plugins wurden deaktiviert!"
 COM_PLUGINS_N_ITEMS_UNPUBLISHED_1="Das Plugin wurde deaktiviert!"
 COM_PLUGINS_NAME_HEADING="Pluginname"
 COM_PLUGINS_NO_ITEM_SELECTED="Kein Plugin ausgewählt"
+COM_PLUGINS_OPTION_ELEMENT="- Element wählen -"
```

Comment:
Hmm bis jetzt wurde "Element" als "Datei" übersetzt:

![image](https://user-images.githubusercontent.com/2596554/44923239-f46f0580-ad47-11e8-8b98-72545b4dae1b.png)

Reply:
Datei macht hier aber keinen Sinn.

![33513771-aab94b64-d749-11e7-9668-ea1531b47205](https://user-images.githubusercontent.com/1752513/44924556-ea4f0600-ad4b-11e8-8260-a16ec02bd05e.png)

Reply:
Stimmt dann ändern wir Datei in Element also auch in den Headings etc.

Reply:
Dann aber für 3.8.13v1 als eigener PR, oder @zero-24

Reply:
Da es keine 3.8.13 geben soll können wir das auch hier rein packen sind ja "nur" fünf strings.

Ich sehe gerade Element wird auch als "Plugindatei(name)" übersetzt: https://github.com/joomlagerman/joomla/blob/76125bab649c609405f8e65373a48fb72095e8bc/administrator/language/de-DE/de-DE.com_plugins.ini#L13-L14 und das würde ja wieder passen?

Also ggf. die Strings auf Plugindatei vereinheitlichen?

Reply:
Ja find ich gut.